201-1-vm-loadbalancer-2-nics/azuredeploy.json

```diff
@@ -0,0 +1,305 @@
+﻿{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "storageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of storage account"
+            }
+        },
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "Admin username"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Admin password"
+            }
+        },
+        "dnsNameforLBIP": {
+            "type": "string",
+            "metadata": {
+                "description": "DNS for Load Balancer IP"
+            }
+        },
+        "vmNamePrefix": {
+            "type": "string",
+            "defaultValue": "myVM",
+            "metadata": {
+                "description": "Prefix to use for VM names"
+            }
+        },
+        "imagePublisher": {
```

Comment:
Since this a template about multi-NICs, all these image parameters can become variables so the user doesn't have to enter many parameters to get this going